### DISCUSSION TITLE
Should WEMI-linking properties be required?

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/sinopia_maps#130] [decided]
Currently WEMI-linking properties are required in Sinopia RDA/RDF templates. However, this means catalogers much use a workaround in order to save work, entering fake IRIs for (related) resource B, so that they can save resource A, add IRI for resource A to resource B, save resource B, then go back to resource A and replace fake with actual IRI.

**PROS AND CONS**
| PRO | CON |
|---|---|
| Setting properties as required in templates aligns with RDA Toolkit [guidance on minimum description of a resource entity](https://access.rdatoolkit.org/en-US_ala-f4fb013b-b64e-37b1-902c-561d27b1e832) | Setting properties as required makes a workaround neccessary and increases complexity of cataloging |
| Setting properties as required ensures that relationships between WEMI resources will be created | Because fake IRIs must be entered at some point initially in order to save a resource and proceed with description, false relationships to nonexistent entities may be created |
| ... | ...|

**Comment by briesenberg07** [comment]
Feedback from 2023-08-15 meeting
CC @AdamSchiff @JianPLee 

> The need to use and then replace fake URLs is *horrible!*
> It's a recipe for copy-paste disasters!
> Slows cataloging down
> Annoying to have to do this for each W,E,M,I

**Reply by briesenberg07** [reply]
DECISION:
- configure default 'https://FAKE_EXPRESSION_REPLACE_ME' for each WEMI-linking property
- add PT remark like 'REMEMBER to replace the filler IRI for [W/E/M/I] once IRI is available!'
- Tag issue https://github.com/uwlib-cams/sinopia_maps/issues/145 in commits

**Comment by AdamSchiff** [comment]
Sounds good to me.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________
